Source/Launcher/General/Generall.cs

```diff
@@ -14,13 +14,14 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 using System.Windows.Forms;
+using CodeImp.Bloodmasters.Launcher.Interface;
 using CodeImp.Bloodmasters.Net;
 using Microsoft.Win32;
 using SharpDX.Direct3D9;
 
-namespace CodeImp.Bloodmasters.Launcher;
+namespace CodeImp.Bloodmasters.Launcher.General;
 
-public class General
+public class Generall
```

Comment:
This is a very strange naming. I suggest improving this.

Let's name this as `Program` (a well-established convention), and also move every class out of `General` namespace to just `CodeImp.Bloodmasters.Launcher` (and to the corresponding file system folder).

This way, `General` will be no more.

Reply:
complete